screens/SendSetupScreen.js

```diff
@@ -352,7 +459,7 @@ const SendSetupScreen = ({
           : 0
       );
     }
-  }, [sendAmountNumber, amountType, fiatRate]);
+  }, [sendAmountCrypto, sendAmountNumber, amountType, fiatRate]);
```

Comment:
`sendAmountCrypto` isn't used in the effect, is it a newly required dependency of it?